examples/azure-aci/Main.scala

```diff
@@ -0,0 +1,44 @@
+import besom.*
+import besom.api.azurenative
+
+@main def main = Pulumi.run {
+  val resourceGroup = azurenative.resources.ResourceGroup("resourceGroup")
+
+  val imageName = "mcr.microsoft.com/azuredocs/aci-helloworld"
+
+  val containerGroup = azurenative.containerinstance.ContainerGroup(
+    name = "containerGroup",
+    azurenative.containerinstance.ContainerGroupArgs(
+      resourceGroupName = resourceGroup.name,
+      osType = azurenative.containerinstance.enums.OperatingSystemTypes.Linux,
+      containers = List(
+        azurenative.containerinstance.inputs.ContainerArgs(
+          name = "hello-world",
+          image = imageName,
+          ports = List(azurenative.containerinstance.inputs.ContainerPortArgs(port = 80)),
+          resources = azurenative.containerinstance.inputs.ResourceRequirementsArgs(
+            requests = azurenative.containerinstance.inputs.ResourceRequestsArgs(
+              cpu = 1.0,
+              memoryInGB = 1.5
+            )
+          )
+        )
+      ),
+      ipAddress = azurenative.containerinstance.inputs.IpAddressArgs(
+        ports = List(
+          azurenative.containerinstance.inputs.PortArgs(
+            port = 80,
+            protocol = azurenative.containerinstance.enums.ContainerGroupNetworkProtocol.TCP
+          )
+        ),
+        `type` = azurenative.containerinstance.enums.ContainerGroupIpAddressType.Public
+      ),
+      restartPolicy = azurenative.containerinstance.enums.ContainerGroupRestartPolicy.Always
+    )
+  )
+
+  Stack(containerGroup).exports(
+    // TODO uncomment when bug https://github.com/VirtusLab/besom/issues/432 will be fixed
+    // containerIPv4Address = containerGroup.ipAddress.ip
```

Comment:
@lbialy would you be able to provide the Output.recover to enable a workaround here?

Reply:
While I agree that we should add recover, I don't really see wow would recover help here. All it will do is allow user to substitute the broken property with something provided by the user.

Reply:
Since the `ip` itself is present and correct, the idea was to workaround this, by ignoring the exception and getting the value.